docs/sources/flow/reference/components/loki.process.md

```diff
@@ -937,6 +947,597 @@ stage {
+
+
+The valid `action` values are `inc`, `dec`, `set`, `add`, or `sub`.
+`inc` and `dec` increment and decrement the metric's value by 1 respectively.
+If `set`, `add, or `sub` is chosen, the extracted value must be convertible
+to a positive float and is set, added or subtracted to the metric's value.
+
+
+#### histogram block
+Defines a histogram metric whose values are recorded in predefined buckets.
+
+
+The following arguments are supported:
+
+Name            | Type          | Description | Default | Required
+--------------- | ------------- | ----------- | ------- | --------
+`name`          | `string`      | The metric name. | | yes
+`buckets`       | `list(float)` | The action to take. Valid actions are `set`, `inc`, `dec`,` add`, or `sub`. | | yes
+`description`   | `string`      | The metric's description and help text. | `""` | no
+`source`        | `string`      | Key from the extracted data map to use for the metric. Defaults to the metric name. | `""` | no
+`prefix`        | `string`      | The prefix to the metric name. | `"loki_process_custom_"` | no
+`idle_duration` | `duration`    | Maximum amount of time to wait until the metric is marked as 'stale' and removed. | `"5m"` | no
+`value`         | `string`      | If set, the metric only changes if `source` exactly matches the `value`. | `""` | no
+
+#### metrics behavior
+
+If `value` is not present, all incoming log entries match.
+
+Label values on created metrics can be dynamic, which can cause exported
+metrics to explode in cardinality, or go stale, for example when a stream stops
+receiving new logs. To prevent unbounded growth of the /metrics endpoint, any
+metrics which have not been updated within `idle_duration` are removed. The
+`idle_duration` must be greater or equal to `"1s"`, and it defaults to `"5m"`
+
+The metric values extracted from the log data are internally converted to
+floats. The supported values are the following:
+
+* integers, floating point numbers
+* string - two types of string formats are supported:
+    * strings that represent floating point numbers: e.g., "0.804" is converted to 0.804.
+    * duration format strings. Valid time units are “ns”, “us”, “ms”, “s”, “m”, “h”. Values in this format are converted as a floating point number of seconds. E.g., "0.5ms" is converted to 0.0005.
+* boolean:
+    * true is converted to 1
+    * false is converted to 0
+
+The following pipeline creates a counter which increments every time any log line is received by using the `match_all` parameter, plus a second counter which adds the byte size of these log lines by using the `count_entry_bytes` parameter.
+
+These two metrics disappear after 24 hours, if no new entries are received, to avoid building up metrics which no longer serve any use. These two metrics are a good starting point to track the volume of log streams in both the number of entries and their byte size, to identify sources of high-volume or high-cardinality data.
```

Comment:
```suggestion
These two metrics disappear after 24 hours if no new entries are received, to avoid building up metrics which no longer serve any use. These two metrics are a good starting point to track the volume of log streams in both the number of entries and their byte size, to identify sources of high-volume or high-cardinality data.
```